lm_eval/models/gpt2.py

```diff
@@ -12,6 +40,11 @@ def __init__(
         subfolder=None,
         tokenizer=None,
         batch_size=1,
+        accelerate=False,
+        device_map=None,
+        max_memory=None,
```

Comment:
This should be exposed in the CLI.

Reply:
And should be named `max_memory_per_gpu` probably.